### PR TITLE
Integrate sonata admin classes and configuration

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
fix https://github.com/symfony-cmf/block-bundle/issues/257

ToDo:
* [x] Add Admin classes
* [x] Add configuration
* [x] Add Configuration tests
* [x] Add Web tests
* [x] Add dependencies
* [x] tests green

### CODE REVIEW

**Comment by ElectricMaxxx** [comment]
I currently uses the way of passing classes as configured parameters into the services, but i don't think that this is usall atm.
What to do now?
* use defaultNull() and overwrite class-values and arguments for not null?
* completely remove the parameters and use FQCN instead - no configuration für classes anymore

**Comment by ElectricMaxxx** [comment]
uhh, this had been the first time, that i had to merge and use the cleared block-bundle to get stuff working in here. Had problems with the old admin classes. But the testsare green now locally.

**Comment by ElectricMaxxx** [comment]
@symfony-cmf/contributors ready to review and to discuss about the the class-parameter situation mentioned above.

File: src/Admin/Block/Imagine/ImagineBlockAdmin.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        parent::configureFormFields($formMapper);
+
+        // image is only required when creating a new item
+        // TODO: sonata is not using one admin instance per object, so this doesn't really work - https://github.com/symfony-cmf/BlockBundle/issues/151
```
Comment (dbu) [inline src/Admin/Block/Imagine/ImagineBlockAdmin.php]: is this a new todo or just copy-paste?

**Pull request merged**